.github/workflows/docker.yml

```diff
@@ -0,0 +1,59 @@
+name: Docker build & push
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - "*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install docker-compose
+        run: sudo apt update && sudo apt install -y docker-compose
+      - name: Generate version.json
+        shell: bash
+        run: |
+          printf '{"commit":"%s","version":"%s","source":"https://github.com/%s","build":"%s/%s/actions/runs/%s"}\n' \
+          "$GITHUB_SHA" \
+          "$GITHUB_REF_NAME" \
```

Comment:
Regarding the use of `GITHUB_REF_NAME` as the version, I want to point out that we currently use `latest` as a tag when things are based on the master branch. See: https://github.com/mozilla/bugbug/blob/359af8d15c0d8ff04a539dc515326b559997259b/.taskcluster.yml#L175-L178